ebssurrogate/scripts/surrogate-bootstrap.sh

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env bash
```

Comment:
why do we need this file / where is it being called from? I don't see any references to it...

Reply:
You're right, I have rebased on develop, but this file hung around.

Reply:
resolved by deleting in latest commit as file had been deleted in `develop` branch already